src/components/verification-grid/verification-grid.ts

```diff
@@ -893,49 +866,161 @@ export class VerificationGridComponent extends AbstractComponent(LitElement) {
     }
   }
 
-  private setDecisionDisabled(disabled: boolean): void {
-    const decisionElements = this.decisionElements ?? [];
-    for (const decisionElement of decisionElements) {
-      decisionElement.disabled = disabled;
+  private addTileDecision(tile: VerificationGridTileComponent, decision: IDecision): void {
+    // during a verification task, we only want the user to make one
+    // verification decision about a tile
+    if (decision instanceof Verification) {
+      const existingVerification = tile.model.verification;
+
+      if (existingVerification) {
+        tile.removeDecision(existingVerification);
+      }
+    } else if (decision instanceof Classification) {
+      // during a classification task, we only want the user to make one
+      // decision about each tag
+      const exclusiveToTags = this.getDecisionsByTag(decision.tag.text);
+
+      // remove all the exclusive tags from the tile
+      for (const exclusiveTag of exclusiveToTags) {
+        if (!exclusiveTag.decisionModel) {
+          continue;
+        }
+
+        tile.removeDecision(exclusiveTag.decisionModel);
+      }
     }
 
-    this.decisionsDisabled = disabled;
+    tile.addDecision(decision);
```

Comment:
this logic feels out of place. I think some of it belongs on the subject wrapper. I think the only part of this that should remain in here is `tile.addDecision`. Modifying the state of the subject wrapper should be the responsibility of the subject wrapper. 

I also think you can simplify the replacement logic if you used a dictionary/map on the subject wrapper to store decisions. e.g. `subject.deicsions[verificationSymbol] = decision`. You could split storage of classifications/verifications on the subject wrapper too.

I don't understand the `exclusiveToTags` part of the code. Is this a toggle implementation?